source/tutorial/autocomplete-component.md

```diff
@@ -38,9 +38,9 @@ In this case we are passing, or "yielding", our filter data to the inner markup
 
 {{#list-filter
    filter=(action 'filterByCity')
-   as |rentals|}}
+   as |filterResults|}}
```

Comment:
what about `filteredRentals`?

Reply:
Sure, that works. I was thinking of it as the "results of the filter", but I can see how that might be mistaken for an action of some kind.

Reply:
Looking back on this, I see I misread this as `filteredResults` instead of `filteredRentals`. Since this was already pulled in to super-rentals like this, would you be alright with keeping it as `filteredResults`? If not, I'll open a new PR to fix it there.